examples/spi/src/app.xc

```diff
@@ -4,6 +4,7 @@
 #include <stdint.h>
 #include <assert.h>
 #include "control.h"
+#include "control_transport_shared.h" // Needed for SPI_DATA_MAX_BYTES
```

Comment:
control_transport_shared.h is not in the api directory so I would not expect it to be used outside of the library

Reply:
I agree with this point, taught the same, the thing is the rest of protocols limits are as well in control_transport_shared.h and used in tests/examples. So my proposal here is leave like this and open an issue to change all the rest, or hardcode here the max and avoid the header. let me know

Reply:
I like option 1, making a jira is a good idea

Reply:
opened: https://xmosjira.atlassian.net/browse/LSM-146?atlOrigin=eyJpIjoiZTYxZjg1YTUyNTlhNGRmMmFmMjRkMzM2MDIyZjExZDUiLCJwIjoiaiJ9